contracts/ArrowVestingBase.sol

```diff
@@ -1,8 +1,8 @@
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.11;
 
 // SPDX-License-Identifier: MIT
 
-import "@openzeppelin-upgradeable/contracts/finance/VestingWalletUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/finance/VestingWalletUpgradeable.sol";
```

Comment:
This change is breaking the existing CI tests, because brownie can no longer find the correct import path. Can we revert this? If not, we'll need to change the brownie import logic to cope with the change.

Reply:
Brownie imports from Github releases, so we should probably just update the path substitution Brownie does to match the NPM package. You can find it in `brownie-config.yaml`.

Reply:
Is this more sound than following official npm releases? I am trying to replicate this in hardhat overrides and now thinking to myself if npm is the better path... allows for better composability IMO

Reply:
I don't see the difference between importing the official npm releases or the offical GitHub releases, since they're both ultimately controlled by OpenZeppelin.

It seems like the prevailing wind is to entirely replace Brownie with Hardhat though, so it makes sense to use whatever works best with Hardhat's configuration and go from there. I'm happy with you just going with the npm import and not bend over backwards to support both frameworks, as long as the test coverage remains the same.